frontend/src/pages/projects/ProjectDetailsContext.tsx

```diff
@@ -48,7 +49,7 @@ type ProjectDetailsContextType = {
   inferenceServices: ContextResourceData<InferenceServiceKind>;
   serverSecrets: ContextResourceData<SecretKind>;
   projectSharingRB: ContextResourceData<RoleBindingKind>;
-  groups: ContextResourceData<GroupKind>;
+  groups: WatchK8sResult<GroupKind[]>;
```

Comment:
This change seems inconsistent with the other properties other than `ContextResourceData` also includes a `refresh` function. `WatchK8sResult` is an array while `ContextResourceData` is an object with named properties.

Can you make a similar type to `ContextResourceData` without the refresh action.